zhaquirks/sinope/switch.py

```diff
@@ -326,6 +327,67 @@ class SinopeTechnologiesLoadController(CustomDevice):
     }
 
 
+class SinopeTechnologiesNewLoadController(CustomDevice):
```

Comment:
IMHO bad naming.
I would prefer `SinopeTechnologiesLoadController _var02` or similar.
Otherwise you can get sometime a `SinopeTechnologiesNewNewLoadController` 😉